box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/InviteCollaboratorsFragment.java

```diff
@@ -0,0 +1,276 @@
+package com.box.androidsdk.share.fragments;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.MultiAutoCompleteTextView;
+import android.widget.Toast;
+
+import com.box.androidsdk.content.BoxException;
+import com.box.androidsdk.content.BoxFutureTask;
+import com.box.androidsdk.content.models.BoxCollaboration;
+import com.box.androidsdk.content.models.BoxFolder;
+import com.box.androidsdk.content.models.BoxSession;
+import com.box.androidsdk.content.models.BoxUser;
+import com.box.androidsdk.content.requests.BoxRequestBatch;
+import com.box.androidsdk.content.requests.BoxRequestsShare;
+import com.box.androidsdk.content.requests.BoxResponse;
+import com.box.androidsdk.content.requests.BoxResponseBatch;
+import com.box.androidsdk.content.utils.BoxLogUtils;
+import com.box.androidsdk.content.utils.SdkUtils;
+import com.box.androidsdk.internal.BoxApiInvitee;
+import com.box.androidsdk.internal.BoxInviteeResponse;
+import com.box.androidsdk.share.CollaborationUtils;
+import com.box.androidsdk.share.R;
+import com.box.androidsdk.share.adapters.InviteeAdapter;
+import com.box.androidsdk.share.internal.BoxListInvitees;
+
+import java.net.HttpURLConnection;
+import java.util.Locale;
+
+public class InviteCollaboratorsFragment extends BoxFragment implements View.OnClickListener, CollaborationRolesDialog.OnRoleSelectedListener {
+
+    protected static final String TAG = InviteCollaboratorsFragment.class.getName();
+    public static final String EXTRA_ACCESS_TOKEN = "InviteCollaboratorsFragment.ExtraAccessToken";
+    private Button mRoleButton;
+    private MultiAutoCompleteTextView mAutoComplete;
+    private InviteeAdapter mAdapter;
+    private BoxCollaboration.Role mSelectedRole;
+    private BoxCollaboration.Role[] mRoles;
```

Comment:
Is there any reason we aren't using just an arraylist instead of simple array for these Roles?